File: desafio1/desafio1.java
class Produto {

    String nome;
    int preco_unitario;
    int quantidade_estoque;

    public Produto (String nome, int preco_unitario, int quantidade_estoque) {
        this.nome = nome;
        this.preco_unitario = preco_unitario;
        this.quantidade_estoque = quantidade_estoque;
    }

    public void exibir() {
        System.out.println(this.nome+ " " + this.preco_unitario + " " + this.quantidade_estoque);
    }

    public int valor_total_estoque () {
        return preco_unitario*quantidade_estoque;
    }

}

public class desafio1 {
    public static void main(String[] args) {
        Produto agua = new Produto("agua",10,30);
        Produto arroz = new Produto("arroz",4,60);
        Produto notebook = new Produto("notebook",10,30);

        agua.exibir();
        System.out.println(agua.valor_total_estoque());
    }
}

File: desafio1/desafio1.py
class Produto:
    def __init__(self, nome, preco_unitario, quantidade_estoque):
        self.nome = nome
        self.preco_unitario = preco_unitario
        self.quantidade_estoque = quantidade_estoque

    def __str__(self):
        return f"{self.nome}({self.preco_unitario} * {self.quantidade_estoque})"

    def valor_total_estoque(self):
        return self.preco_unitario * self.quantidade_estoque



agua = Produto('agua',10,30)
arroz = Produto('arroz',4,60)
notebook = Produto('notebook',10,30)

print(agua)
print(agua.valor_total_estoque())

